### PR TITLE
Suppress sporadic exceptions from JumpList creation code

### LINKED ISSUE #9295: JumpList causes pwsh to fail
After upgrading to PowerShell 6.2, from 6.1.3, and removing PowerShell 6 Preview (6.2 RC 1), PowerShell often will not start.  The window appears, powershell looks like its starting, and then the window closes.    Several attempts are usually needed to get a window to successfully get to a prompt.

All the update/uninstall process was done at one time.  I have two different machines that seem to be exhibiting this issue, but I don't have any that are not.

# Environment data

```none
Name                           Value
----                           -----
PSVersion                      6.2.0
PSEdition                      Core
GitCommitId                    6.2.0
OS                             Microsoft Windows 10.0.17763
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
My other machine is running the most recent Windows 19H1 Insiders Preview.

I am also experiencing a lot of crashing in VS Code's PowerShell 2.0.1 extension with certain documents open, but at least PowerShell 6.2 usually opens fine in the 'integrated console' of the extension.  May or may not be related, but I am not seeing anyone posting on that topic either.

Let me know if there is any other data I can include.

Note I originally had trouble with earlier updates to the preview releases, see #8442.  I have not reviewed the condition there as in this case, PowerShell does usually open if I keep trying.

### PR DESCRIPTION
**Pull request by bergmeister**
# PR Summary

Closes #9295
Catch sporadic exceptions from JumpList creation code. Once the jumplist is registered, it persists, therefore no problem to swallow the occasional exception.
Comment [here](https://github.com/PowerShell/PowerShell/issues/9295#issuecomment-503972024) suggest that this fix improves it.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs [hunk 1]
```diff
@@ -193,13 +193,21 @@ internal sealed partial class ConsoleHost
...
+                {
+                    // Creating a JumpList entry takes around 55ms when the PowerShell process is interactive and
+                    // owns the current window (otherwise it does a fast exit anyway). Since there is no 'GET' like API,
+                    // we always have to execute this call because we do not know if it has been created yet.
+                    // The JumpList does persist as long as the filepath of the executable does not change but there
+                    // could be disruptions to it like e.g. the bi-annual Windows update, we decided to
+                    // not over-optimize this and always create the JumpList as a non-blocking background task instead.
+                    Task.Run(() => TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator));
+                }
+                catch
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs]: It does not work as you expect https://stackoverflow.com/questions/32067034/how-to-handle-task-run-exception

Comment (bergmeister) [reply src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs]: Thanks. I did not think of that. Should we have the try catch inside the lambda expression then as we are not async here and also do not want to block.

**Comment by iSazonov** [comment]
I think we can merge the small change and watch some time.

**Comment by bergmeister** [comment]
Comment [here](https://github.com/PowerShell/PowerShell/issues/9295#issuecomment-503972024) suggest that this fix improves it, therefore changing PR state from draft to published.

**Review by iSazonov — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
Since this hasn't repro'd for anyone on the PowerShell team, and the change itself is not risky, I agree we should take this for Preview.2 and watch the feedback.

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by iSazonov** [comment]
@PoshChan Please remind me in 12 hours

**Comment by PoshChan** [comment]
@iSazonov, this is the reminder you requested 12 hours ago

**Review by TravisEz13 — COMMENTED** [review]
LGTM

**Comment by daxian-dbw** [comment]
I would love we figure out the root cause and understand why this change fixed the issue. The exception should be captured by the task and save in the `Exception` property, so theoretically, exception shouldn't be a problem unless it's fatal (e.g. `AccessViolationException`)

**Comment by iSazonov** [comment]
@daxian-dbw Is your request to write the exception to log?

**Comment by daxian-dbw** [comment]
@iSazonov That would be helpful to understand what is actually happening.

**Comment by iSazonov** [comment]
@bergmeister Could you please address @daxian-dbw comment?

**Comment by bergmeister** [comment]
When you say 'write to log', do you mean write to console host?

**Comment by TravisEz13** [comment]
@bergmeister EventLog would be better.  Writing the the console would be disruptive.

**Pull request merged**